assets/js/atomic/blocks/product-elements/shared/config.tsx

```diff
@@ -23,12 +25,10 @@ export default {
 		html: false,
 	},
 	parent: isExperimentalBuild()
-		? null
+		? undefined
 		: [ '@woocommerce/all-products', '@woocommerce/single-product' ],
 	save,
-	deprecated: [
-		{
-			save() {},
```

Comment:
**Note to reviewer:** Was this needed? If so can you explain the reason for an empty `save()` here as I removed it in my PR.